azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -419,21 +423,92 @@ public int selectAndUpdateExecutionWithLocking(final int executorId, final boole
     }
   }
 
+  /**
+   * This method is used to select executions in batch. It will apply lock and fetch executions.
+   * It will also
```

Comment:
Incomplete comment. It would also be nice to see short one-liner descriptions against each parameter.

Reply:
Fixed it. Thanks!